test/pylib/scylla_server.py

```diff
@@ -316,6 +320,11 @@ async def stop(self) -> None:
             return
 
         try:
+            if self.control_connection is not None:
+                self.control_connection.shutdown()
+            if self.control_cluster is not None:
+                self.control_cluster.shutdown()
```

Comment:
Looks good. Do you know when/why these warnings appear?
I'm curious because in cql-pytest I didn't do this shutdown() when tearing down the cql fixture - but didn't see these warnings.

Reply:
There is no asyncio in cql-pytest, they appear only if the reactor is running so the drivers do background work in the current thread.